jira/src/appsscript.json

```diff
@@ -0,0 +1,16 @@
+{
+  "exceptionLogging": "STACKDRIVER",
+  "dataStudio": {
+    "name": "Jira Community Connector",
```

Comment:
Please remove Community Connector from the name. Maybe `Jira Issues`. Since users will see this in a section about community connectors, it's a bit redundant.